workers/unity/Assets/Gdk/Core/Worker/WorkerBase.cs

```diff
@@ -61,6 +61,18 @@ public bool Connect(ConnectionConfig config)
             return true;
         }
 
+        public void Disconnect()
+        {
+            if (Connection == null)
+            {
+                return;
+            }
+
+            View.Disconnect("Disconnect called.");
```

Comment:
Please change this to something which makes it clear that it was disconnected internally. As a user seeing this while debugging will be confusing.

Reply:
Updated, should hopefully be better,